src/pages/AddProject.tsx

```diff
@@ -231,8 +231,13 @@ class AddProject extends React.Component<RouteComponentProps, FormState> {
         </div>
 
         {(this.state.serverError.message) && <div className="error-message">
-          <p>{this.state.serverError.message}</p>
```

Comment:
Let's make sure to keep a server-side error message on L234!

Reply:
okay